main.js

```diff
@@ -0,0 +1,4 @@
+const ONE = 1;
```

Comment:
'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).